ui/raidboss/data/07-dt/raid/r5s.ts

```diff
@@ -270,7 +278,72 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     {
-      id: 'R5S Wavelength',
+      // Wavelength α debuff timers are applied with 40.5, 25.5, 25.5, 30.5 or
+      //  38.0, 23.0, 23.0, 28.0 durations depending on which group gets hit first
+      //
+      // Wavelength β debuff timers are applied with 45.5, 30.5, 20.5, 25.5 or
+      //  43.0, 28.0, 18.0, 23.0 durations depending on which group gets hit first
+      id: 'R5S Wavelength Merge Order',
+      type: 'GainsEffect',
+      netRegex: { effectId: ['116E', '116F'] },
+      preRun: (data, matches) => {
+        matches.effectId === '116E' ? data.wavelengthCount.alpha++ : data.wavelengthCount.beta++;
+      },
+      durationSeconds: (_data, matches) => parseFloat(matches.duration),
```

Comment:
Should this be `parseFloat(matches.duration) - 5` so that it disappears when `Merge Reminder` fires?

Reply:
They're different severity levels so I don't know if it matters.